src/org/traccar/api/resource/ServerResource.java

```diff
@@ -47,4 +48,10 @@ public Response update(Server entity) throws SQLException {
         return Response.ok(entity).build();
     }
 
+    @Path("geocoding")
```

Comment:
Have not found better place for now.
May be some day we will implement some "Utility" endpoint

Reply:
I think we can rename path to `geocode`.